roles/ffh.yanic/files/apply_patches.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cleanup() {
+	git am --abort
+	git reset --hard origin/master
+	>&2 echo "Warning: git am of $file failed"
+	>&2 echo "git am --abort has already been executed"
+	>&2 echo "as well as 'git reset --hard origin/master'"
+	>&2 echo "to reset manually, remove the patches directory'"
+	exit 1
+}
+trap 'cleanup' ERR
+
+echo > aiyion
```

Comment:
;-)

Reply:
sorry, gotta remove soon, was not intended.

Reply:
resolved in 62ab498c60ed1e74a97e71e011d0bdac139ebafb